### PR TITLE
tcp patches

### PR DESCRIPTION
**Pull request by Rasenkai**

### CODE REVIEW

File: etc/sysctl.d/99-cachyos-settings.conf [hunk 1]
```diff
@@ -74,6 +74,21 @@ net.ipv4.tcp_congestion_control = bbr
 # Helps protect against SYN flood attacks. Only kicks in when net.ipv4.tcp_max_syn_backlog is reached: 
 net.ipv4.tcp_syncookies = 1
 
+# TCP Enable ECN Negotiation by default
+net.ipv4.sysctl_tcp_ecn = 1
```
Comment (Torston420) [inline etc/sysctl.d/99-cachyos-settings.conf:78]: Are you sure you didn't mean to write `net.ipv4.tcp_ecn`?

**Comment by ventureoo** [comment]
We had previously experienced some regressions with sysctl parameters for the network, so we gave up on them. But this set doesn't seem to be that large, so we can test this and then decide which parameters are okay. I think @array is more compotent on the question, which is why I listed him as a reviewer.

**Comment by aarrayy** [comment]
Should be fine but afaik TCP timestamping is needed for some of the BBRv3 functionality, if you take a look into their source code, they're some references about timestamps and RTT calculations and if i'm not wrong net.ipv4.tcp_timestamps = 0 could make it worse?, besides this. The other three are fine and could be added

**Review by aarrayy — COMMENTED** [review]
This could affect BBRv3

**Review by aarrayy — APPROVED** [review]

**Comment by ptr1337** [comment]
Please sign your commits @Rasenkai

**Pull request merged**